src/AMSlib/ml/hdcache.hpp

```diff
@@ -17,6 +17,7 @@
 #include <stdexcept>
 #include <string>
 #include <type_traits>
+#include <unordered_map>
```

Comment:
unused?

Reply:
We use an unordered map. It is likely it was pulling the header from another include.